labs-ios-starter/Application/View/UI/Custom Objects/CircularButton.swift

```diff
@@ -0,0 +1,100 @@
+// Copyright © 2020 Lambda, Inc. All rights reserved.
+// CircularButton.swift
+
+import UIKit
+
+/// The system icons used throughout the app, enums prevents typos -> UIImage(systemName: "[HERE]")
+enum Icon: String {
+    case plus = "plus"
+}
+
+/// A rounded button with an ICON
+class CircularButton: UIButton {
+    
+    // MARK: - Initializers
+    
+    /// The initializer called directly by the Developer
+    convenience init(withIcon systemName: Icon?, size: CGFloat?) {
+        self.init()
```

Comment:
Could add `translatesAutoresizingMaskIntoConstraints = false` here - I think you may have on call earlier